src/cephadm/cephadm

```diff
@@ -1239,6 +1239,10 @@ def deploy_daemon_units(fsid, uid, gid, daemon_type, daemon_id, c,
             f.write(' '.join(poststop.run_cmd()) + '\n')
         os.fchmod(f.fileno(), 0o600)
 
+    with open(data_dir + '/unit.image', 'w') as f:
```

Comment:
too bad there is not an easy way to simply source this in as the image arg for unit.run